Stratis.FederatedSidechains.AdminDashboard.csproj

```diff
@@ -20,4 +27,422 @@
     <PackageReference Include="RestSharp" Version="106.6.9" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
I'll remove this clutter, leftovers from trying to move wwwroot to ClientApp folder and let the bundler build to wwwroot, sadly that doesn't move fonts/images so a different packer is needed if you want that, for now I just added build output to the gitignore so it doesn't get added to changes.